doc/lua_api.txt

```diff
@@ -6946,13 +6946,16 @@ object you are working with still exists.
             (default: `true`)
         * `texture`: A regular texture for the sun. Setting to `""`
             will re-enable the mesh sun. (default: "sun.png", if it exists)
+            At sunrise, the texture appears non-rotated.
         * `tonemap`: A 512x1 texture containing the tonemap for the sun
             (default: `"sun_tonemap.png"`)
         * `sunrise`: A regular texture for the sunrise texture.
             (default: `"sunrisebg.png"`)
         * `sunrise_visible`: Boolean for whether the sunrise texture is visible.
             (default: `true`)
         * `scale`: Float controlling the overall size of the sun. (default: `1`)
+            Note: For equal `scale` values, the sun is bigger than the moon by a
+            factor of `0.119/0.076`.
```

Comment:
```suggestion
            Note: For legacy reasons, the sun is bigger than the moon by a
            factor of `0.119/0.076`.
```

Reply:
Done.
But I've kept the "for equal `scale` values" because one might otherwise read it as "the sun is always bigger than the moon by this factor. if you set the one scale, the other one is automatically adjusted to keep this fact", which is wrong. Instead, one can set both scale values independently.